lib/bottom_navy_bar.dart

```diff
@@ -157,7 +159,9 @@ class _ItemWidget extends StatelessWidget {
                   data: IconThemeData(
                     size: iconSize,
                     color: isSelected
-                        ? item.activeColor.withOpacity(1)
+                        ? item.solidBackground
+                            ? Colors.white
+                            : item.activeColor.withOpacity(1)
```

Comment:
I'm a bit confused, the property is called `solidBackground` but it changes the color of the icon as well. You can either rename the property name to something like `useSolidColors` or revert this change.